src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -80,70 +89,184 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.ubuntu_pro_client_all_updates_cached = []
         self.ubuntu_pro_client_all_updates_versions_cached = []
         
-        self.package_install_expected_avg_time_in_seconds = 90 # As per telemetry data, the average time to install package is around 81 seconds for apt.
+        self.package_install_expected_avg_time_in_seconds = 90  # As per telemetry data, the average time to install package is around 81 seconds for apt.
```

Comment:
nit: why extra space?

Reply:
![image](https://github.com/user-attachments/assets/726e3915-942d-46ff-a1a2-2c257e9e19a8)

Reply:
**[Candidate for resolving]** IDE reason provided.